tools/ingest-converter/src/main/resources/ingest-gsub.js

```diff
@@ -0,0 +1,28 @@
+var IngestGsub = {
+    has_gsub: function (processor) {
+        return !!processor["gsub"];
+    },
+    gsub_hash: function (processor) {
+        var gsub_data = processor["gsub"];
+        return IngestConverter.create_field(
+            "gsub",
+            "[\n" + [gsub_data["field"], gsub_data["pattern"], gsub_data["replacement"]]
```

Comment:
@suyograo now that you say it, it's not only "can" I think ... because the naming has to potentially be translated from dots to `[]` for `gsub_data["field"]` right?